api/api/settings.py

```diff
@@ -75,8 +76,13 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "NAME": "bonedata",
+        "USER": "brukegetachew",
+        "PASSWORD": "",
```

Comment:
we should remove the  "USER" line. It should default the local user and "just work" locally for those of us not named brukegetachew. Also empirically for me removing PASSWORD also works.